### ISSUE TITLE
Resnet Throws Error

### ISSUE THREAD

**Issue by craterkamath** [AlexeyAB/darknet#4736]
I'm trying to train ResNet152_YOLO on my custom data.

I have downloaded a CFG from this link:
https://drive.google.com/drive/folders/1rof0f8mCNKTYJOCWElTYQLOq513Gh5_V

I have altered the Input width and height to 256.
I have 4 classes: I've changed the [convolutional] and [region] section also.

The program shows corresponding error. I tried using another cfg file also, It didnt change the output.

![resnet_error](https://user-images.githubusercontent.com/23616388/72870873-55b5c100-3d0f-11ea-820c-0663288c402a.PNG)

**Comment by craterkamath** [comment]
https://github.com/pjreddie/darknet/issues/187 Talks about how the darknet has resizing issues, I thought that was resolved in a later update.

**Comment by AlexeyAB** [comment]
New [shortcut] layer supports multiinput, weights and normalization, but doesn't support different width & height of input layers.

I will think how can I fix it.

**Comment by craterkamath** [comment]
Update: I tried running the same cfg using the native darknet repo pjreddie/darknet, it works just fine.

**Comment by AlexeyAB** [comment]
Yes, **because new `[shortcut]` layer** supports multiinput, weights and normalization, but **doesn't support different width & height of input layers.**

**Comment by craterkamath** [comment]
@AlexeyAB  Any updates on this issue?

**Comment by zunairaR** [comment]
I am having the same problem @craterkamath and @AlexeyAB . Can anyone help???
